Userland/Libraries/LibTLS/Record.cpp

```diff
@@ -382,7 +382,7 @@ ssize_t TLSv12::handle_message(ReadonlyBytes buffer)
     case MessageType::ChangeCipher:
         if (m_context.connection_status != ConnectionStatus::KeyExchange) {
             dbgln("unexpected change cipher message");
-            auto packet = build_alert(true, (u8)AlertDescription::UnexpectedMessage);
+            [[maybe_unused]] auto packet = build_alert(true, (u8)AlertDescription::UnexpectedMessage);
```

Comment:
pls `write_packet()` that (or I could do that after this is merged, either is fine)
This fix was part of the IODevice PR that got nowhere 😅

Reply:
Cool, added. :)